### ISSUE TITLE
[Feature request] Save/load filter graph to/from file

### ISSUE THREAD

**Issue by christopher317** [antiboredom/ffmpeg-explorer#32]
The moment you close or reload the page, everything you did with the filter graph is gone, hence the need to be able to save the current state of the filter graph to a file so you can continue working on it even if you need to close or reload the page.

I'm a little surprised no one else thought of this, if its already possible please let me know how, its not obvious how to save the current graph just by looking at it.

**Comment by antiboredom** [comment]
Yes - saving state between reloads would be a good feature! The functionality is almost already built in, would basically just have to add the UI for it. You can actually already save the graph (cmd-s) but there's no way to load one in yet...

**Comment by abdelrahman-tarek-0** [comment]
@antiboredom 
can you take a look at my pull request

**Comment by hsngerami** [comment]
@christopher317 I recommend use [VideoAlchemy](https://github.com/viddotech/videoalchemy) , so you don't need to worry about miss or save states!
you can write more readble ffmpeg filter graph in a yaml.

**Comment by inkstray** [comment]
Could there be a possibility for an export button? (I can't use CTRL+S on Android)
